src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseController.java

```diff
@@ -38,6 +38,7 @@
  */
 @Slf4j
 @RequiredArgsConstructor
+@SuppressWarnings({"PMD.TooManyMethods"})
```

Comment:
Why adding this suppression with no other code? if it really needs to be in, check if we're supressing globally (because I think we are, at least for callbacks)

Reply:
Ah, I actually moved Bundles to a new controller so I could remove this suppression...I just forgot to remove it. Removed now :)